src/manifest.rs

```diff
@@ -1,13 +1,17 @@
 // (c) Copyright 2023 Helsing GmbH. All rights reserved.
 
+use std::path::Path;
+use std::{collections::HashMap, fmt};
+
 use eyre::Context;
 use semver::{Version, VersionReq};
 use serde::{Deserialize, Serialize};
-use std::{collections::HashMap, fmt};
 use tokio::fs;
 
 use crate::package::{PackageId, PackageType};
 
+// TODO(rfink): For symmetry between the vendored and the local packages, we should consider
```

Comment:
worth discussion, @mara214 . I find the asymmetry between the local and the vendored packages a bit strange